site/src/routes/layout.css

```
@@ -0,0 +1,257 @@
+@import "tailwindcss";
+
+@theme {
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)